src/presenters/package-teaser-presenter.js

```diff
@@ -0,0 +1,61 @@
+'use strict';
+
+const MAX_LIST_CONTENT = 3;
+
+const TeaserPresenter = class TeaserPresenter {
+
+	constructor (data) {
+		this.data = data || {};
+	}
+
+	// returns all top level class names appropriate for the teaser
+	get classModifiers () {
+		const theme = this.data.design.theme;
```

Comment:
Fields like design.theme, contains etc aren't going to be coming through on Front page etc - do we also need GraphQL query blobs for packages?